components/drawer/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react'
+import { Drawer } from '@mui/material'
+import { StyledIcon } from './StyledDrawer'
+import { useDispatch, useSelector } from 'react-redux'
+import { closeDrawer } from '@/lib/redux/slices/drawer'
+import { Close } from '@mui/icons-material'
+
+const FormDrawer = () => {
+    const dispatch = useDispatch()
+    const { content, showing } = useSelector((state: any) => state.drawer)
+
+    return (
+        <Drawer
+            anchor="right"
+            open={showing}
+            PaperProps={{ style: { width: '50%' } }}>
+            <StyledIcon onClick={() => dispatch(closeDrawer())}>
```

Comment:
inside of the styleddrawerheader, we should display the drawer title in a span. Title is sourced by the redux store.

```
<StyledDrawerHeader onClick={}>
  <span>{title}</span>
  <Close className="close-icon" />
</StyledDrawerHeader>
```

Reply:
Obviously, style so the title is styled like an appropriate title, bold and all. Then position the title to the left, close to the right.